src/test/scala/org/broadinstitute/clio/webservice/AuthorizationWebServiceSpec.scala

```diff
@@ -0,0 +1,67 @@
+package org.broadinstitute.clio.webservice
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import akka.http.scaladsl.server.AuthorizationFailedRejection
+import org.scalatest.{FlatSpec, Matchers}
+import org.broadinstitute.clio.model._
+import org.broadinstitute.clio.service._
+
+class AuthWebServiceSpec extends FlatSpec
+    with Matchers with AuthorizationWebService with ScalatestRouteTest {
+
+  behavior of "AuthWebService"
+
+  it should "authorizationRoute" in {
+
+    Get("/authorization").
+      addHeader(OidcClaimExpiresIn(AuthorizationService.mock.expiration)).
```

Comment:
Why is this code using `AuthorizationService.mock` to provide the value for `expiration`? It looks like any value would work here.